src/components/Proposals/SummaryBar.js

```diff
@@ -4,23 +4,34 @@ import { RADIUS, useTheme } from '@aragon/ui'
 import { animated } from 'react-spring'
 import styled from 'styled-components'
 
-function SummaryBar({ positiveSize, negativeSize, requiredSize, ...props }) {
+function SummaryBar({
+  disabled,
+  positiveSize,
+  negativeSize,
+  requiredSize,
+  ...props
+}) {
   const theme = useTheme()
+
   return (
     <Main {...props}>
       <CombinedBar>
         {!!positiveSize && (
           <BarPart
             style={{
-              backgroundColor: theme.positive,
+              backgroundColor: `${
+                disabled ? theme.surfaceOpened : theme.positive
+              }`,
```

Comment:
For consistency perhaps we should use the `css` prop to apply these, rather than `style`.